dev/extra-ml-requirements.txt

```
@@ -18,6 +18,8 @@ tensorflow==1.15.2
 # Required by mlflow.pytorch
 torch==1.6.0
 torchvision==0.5.0
+# Required by mlflow.pytorch
+pytorch_lightning==1.0.2
```

Can we unpin `pytorch_lightning `?

---

@harupy It needs to be ver 1.0.2 and above. The PTL apis have changed and there were break issues with the prior versions. The current PTL api is expected to be more stable going forward.

---

```suggestion
pytorch_lightning>=1.0.2
```

@chauhang Thanks! If so, should we un-max-pin the version?

---

> @chauhang Thanks! If so, should we un-max-pin the version?

updated it to `>=`